management/vcr_test.go

```diff
@@ -0,0 +1,194 @@
+package management
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/dnaeon/go-vcr/v2/cassette"
+	"github.com/dnaeon/go-vcr/v2/recorder"
+	"github.com/stretchr/testify/require"
+
+	"github.com/auth0/go-auth0"
+)
+
+func setupVCR(t *testing.T) {
+	t.Helper()
+
+	if !vcrTestsEnabled {
```

Comment:
This is to ensure that we can also run the tests without the recordings easily. If AUTH0_VCR is false then we skip the entire VCR setup.